docs/teams/jester.html

```diff
@@ -990,6 +1003,66 @@ <h3 style="margin: 10px 0; padding: 0;">Role Configuration:</h3>
+                        <tr>
+                            <td>ttt_sponge_device_time</td>
+                            <td>8</td>
+                            <td>Integer</td>
+                            <td>The amount of time (in seconds) the spongifier takes to use.</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_beggar</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the beggar should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_bodysnatcher</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the bodysnatcher should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_clown</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the clown should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_cupid</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the cupid should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_guesser</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the guesser should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_jester</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the jester should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_lootgoblin</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the lootgoblin should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_shadow</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the shadow should get the spongifier</td>
+                        </tr>
+                        <tr>
+                            <td>ttt_sponge_device_for_swapper</td>
+                            <td>0</td>
+                            <td>Boolean</td>
+                            <td>Whether the swapper should get the spongifier</td>
+                        </tr>
```

Comment:
This could be condensed down since it is automatically generated for all jester roles. We do the same thing for some other ConVars (e.g. `ttt_drunk_can_be_*`)
```suggestion
                        <tr>
                            <td>ttt_sponge_device_for_*</td>
                            <td>0</td>
                            <td>Boolean</td>
                            <td>Whether a specific jester role should get the spongifier. Replace * with the name of the jester role you want to configure without spaces or capital letters.</td>
                        </tr>
```